src/computer_player.c

```diff
@@ -489,8 +489,7 @@ int computer_process_request_card(void)
 {
     int result = 0;
     Card_t draw_card = draw_one_card();
-    printf("Conputer draws a new card from the draw pile \n");
-    printf("No playable card on hand, drawing a new card from deck (%s,%s).\n", CARD_COLOR_STRING[draw_card.color], CARD_NAME_STRING[draw_card.name]);//TODO Remove this line after testing
+    printf("COMPUTER requested a new card, since no playable card on hand. \n");
```

Comment:
"Computer draws a new card from the pile" -- > Can you change to this please?

Reply:
done